composer.json

```diff
@@ -15,7 +15,10 @@
         "symfony-cmf/routing-auto": "^2.0",
         "symfony-cmf/routing-bundle": "^2.0",
         "aferrandini/urlizer": "1.0.*",
-        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0"
+        "phpdocumentor/reflection-docblock": "^3.1, !=3.2.0",
+        "doctrine/orm": "~2.5.0",
```

Comment:
we can't add this as a general dependency, the bundle is supposed to work without doctrine orm.